Translated #14. Ready for proofreading. (#32)
vojtapolasek/Fegora: docs/handout_en.md

alespoň 4 Gb paměti RAM. přidejte počítači CD mechaniku tak, aby její obsah odkazoval na stažený live obraz Fegory. Pokud si chcete Linux opravdu jen vyzkoušet, nemusíte přidávat žádný virtuální pevný disk. Pokud si chcete vyzkoušet i instalaci, přidejte dostatečně velký pevný disk (20 GiB by mělo stačit. Pak už jen stačí spustit
virtuální počítač a Linux by měl automaticky naběhnout.

## I have managed to run Linux. What now?

Congratulations. You have now overcome the most difficult problem
that beginners have with Linux and that may scare them off eventually.
Now, things will go relatively easy. As mentioned before, after
the system has started successfully you will hear the "The screen 
reader is switched on." message.

Press the **Ctrl+Alt+D** keyboard shortcuts which takes you to the 

> workplace. You need to do it after the system starts, otherwise it will 

Reviewer: ```suggestion
desktop. You need to do it after the system starts, otherwise it will 
```

not accept the keyboard shortcuts. The speech volume might be too loud 
for you. You can change the volume of the system sound using the dedicated 
hardware buttons or the **Alt+Win** combination with the **up** and **down** 
arrow keys.

> Now you are on the workplace. Using the arrows, you can browse through several 

Reviewer: ```suggestion
Now you are on the desktop. Using the arrows, you can browse through several 
```

icons. You can activate the icons with the **Enter** key. If you activate 
the *Home liveuser* icon, the file manager opens your home folder. The system 
has already created several folders in this location and will store all your 
settings here.

> On the workplace, there will be a very interesting file called `keyboard_shortcuts.txt`

Reviewer: ```suggestion
On the desktop there will be a very interesting file called `keyboard_shortcuts.txt`
```

which contains a list of useful keyboard shortcuts. Use the arrows to navigate to
this file and press **Enter**. The file will open in a text editor. You will

> probably notice that text editting works as in Windows. You can close the text editor

Reviewer: ```suggestion
probably notice that text editing works as in Windows. You can close the text editor
```

using the **Alt+F4** keyboard shortcut.

> On the workspace, the `handout.html` file is located, too, and you are probably

Reviewer: ```suggestion
The `handout.html` file is located on the desktop as well, and you are probably
```
